### ISSUE TITLE
Avast blocks Blazor Demo Apps

### ISSUE THREAD

**Issue by vectorix** [aspnet/Blazor#491]
Just wanted to report an Avast detection error which ought to be a false alarm.

![avastblazorrealworld](https://user-images.githubusercontent.com/24557849/38371860-13a63cf8-3920-11e8-8d6d-62ab5b57f8d6.png)
![avastflightfinderclient](https://user-images.githubusercontent.com/24557849/38371872-18a64fb8-3920-11e8-822d-e7ed7f01bc28.png)

Avast Program Version: 18.3.2333 build 18.3.3860.0
Virus definition version: 180405-0

**Comment by jamie-lord** [comment]
Yep, I've seen this with Symantech too. Not really anything we can do. I've just excluded Firefox's working directory from active scanning. They don't like the fact that dll files suddenly appear on the system.

**Comment by RyoukoKonpaku** [comment]
Same here blocked on my company's network, the DLLs won't download. Though strangely enough I could load the DLLs from the simple Blazor sample on github, so I assume this is maybe just a site filter from the company firewall. Had no problems yet with my AV at home though, using ESET.

**Comment by SteveSandersonMS** [comment]
Thanks for reporting this! Having extra known test cases will help us when we implement #172.

Closing as duplicate of #172.
